packages/web/src/common/store/pages/profile/lineups/tracks/sagas.js

```diff
@@ -39,80 +38,18 @@ function* getTracks({ offset, limit, payload, handle }) {
   const currentUserId = yield select(getUserId)
   const profileHandle = handle.toLowerCase()
 
-  // Wait for user to receive social handles
-  // We need to know ahead of time whether we want to request
-  // the "artist pick" track in addition to the artist's tracks.
-  // TODO: Move artist pick to chain/discprov to avoid this extra trip
-  const user = yield call(
-    waitForValue,
-    getUser,
-    { handle: profileHandle },
-    (user) => 'twitter_handle' in user
-  )
   const sort = payload?.sort === TracksSortMode.POPULAR ? 'plays' : 'date'
   const getUnlisted = true
-  const getFeatureEnabled = yield getContext('getFeatureEnabled')
-  const readArtistPickFromDiscoveryEnabled = yield call(
-    getFeatureEnabled,
-    FeatureFlags.READ_ARTIST_PICK_FROM_DISCOVERY
-  ) ?? false
 
-  if (!readArtistPickFromDiscoveryEnabled && user._artist_pick) {
```

Comment:
Lineup will already be sorted with artist pick first in the discovery API, which reads the `artist_pick_track_id` in the discovery users table